### PR TITLE
[s] fixes damage multiplicative exploit with foam darts

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

fixes a pretty bad exploit with foam darts

## Why It's Good For The Game

- Modify a dart with a pen, giving it 5 damage
- Target someone's mouth (or a monkeyman)
- Fire
- Firing sequence for handle_suicide multiplies damage by 5 on the projectile instance
- Dropped projectile has 25
- Repeat 1 more time for 125 instacrit dart, or another time after that for 600 instakill dart

all discovery credits go to @necromanceranne, this was brought to my attention

## Changelog
Probably doesn't need one

### CODE REVIEW

**Comment by necromanceranne** [comment]
Pew

**Comment by ATH1909** [comment]
wasn't this already fixed?

**Comment by ATH1909** [comment]
https://github.com/tgstation/tgstation/issues/52999
https://github.com/tgstation/tgstation/pull/52995

**Comment by ATH1909** [comment]
wait

ohhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

File: code/modules/projectiles/projectile/reusable/foam_dart.dm [hunk 1]
```diff
@@ -20,8 +20,9 @@
 	newcasing.modified = modified
 	var/obj/projectile/bullet/reusable/foam_dart/newdart = newcasing.BB
 	newdart.modified = modified
-	newdart.damage = damage
-	newdart.nodamage = nodamage
+	if(modified)
+		newdart.damage = 5
```
Comment (ATH1909) [inline code/modules/projectiles/projectile/reusable/foam_dart.dm:24]: lowkey, I wouldn't mind if this was buffed to be like 10 damage

5 damage from a projectile that you have to specially prepare (with a hard to mass-produce resource, pens) is GARBAGE, even when burst fired

**Comment by carshalash** [comment]
Yeah, this should get merged asap.

File: code/modules/projectiles/projectile/reusable/foam_dart.dm (hunk 1 above)
Comment (silicons) [reply code/modules/projectiles/projectile/reusable/foam_dart.dm:24]: not my problem this is an exploit fix, make the pr after please

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Better PR title please

File: code/modules/projectiles/projectile/reusable/foam_dart.dm (hunk 1 above)
Comment (blessedmulligan) [reply code/modules/projectiles/projectile/reusable/foam_dart.dm:24]: It's not supposed to be a pro gamer move, it's supposed to be slightly malicious and mildly obnoxious.

**Comment by TheVekter** [comment]
@Cyberboss Given that this is marked for speedmerge and a critical fix, wouldn't it be better to just change the title for them and merge it instead of letting it sit for ~15 hours?

**Comment by silicons** [comment]
@Cyberboss done

**Comment by Cyberboss** [comment]
> 
> 
> @Cyberboss Given that this is marked for speedmerge and a critical fix, wouldn't it be better to just change the title for them and merge it instead of letting it sit for ~15 hours?

Professionals have standards

**Pull request merged**